v2/i18n/icons.go

```diff
@@ -64,39 +64,39 @@ func generateGooglePlayStoreIconLink(lang string) string {
-	language.Korean:               generateGooglePlayStoreIconLink("ko"),    // ko
-	language.Dutch:                generateGooglePlayStoreIconLink("nl"),    // nl
-	language.Norwegian:            generateGooglePlayStoreIconLink("no"),    // no
-	language.Polish:               generateGooglePlayStoreIconLink("pl"),    // pl
-	language.Portuguese:           generateGooglePlayStoreIconLink("pt"),    // pt
-	language.BrazilianPortuguese:  generateGooglePlayStoreIconLink("pt-br"), // pt-BR
-	language.EuropeanPortuguese:   generateGooglePlayStoreIconLink("pt"),    // pt-PT
-	language.Russian:              generateGooglePlayStoreIconLink("ru"),    // ru
-	language.Swedish:              generateGooglePlayStoreIconLink("sv"),    // sv
-	language.Thai:                 generateGooglePlayStoreIconLink("th"),    // th
-	language.Turkish:              generateGooglePlayStoreIconLink("tr"),    // tr
-	language.Ukrainian:            generateGooglePlayStoreIconLink("uk"),    // uk
-	language.Vietnamese:           generateGooglePlayStoreIconLink("vi"),    // vi
-	language.Chinese:              generateGooglePlayStoreIconLink("zh"),    // zh
-	language.SimplifiedChinese:    generateGooglePlayStoreIconLink("zh"),    // zh-Hans
-	language.TraditionalChinese:   generateGooglePlayStoreIconLink("zh"),    // zh-Hant
+	language.Arabic:               "ar",    // ar
+	language.ModernStandardArabic: "ar",    // ar-001
+	language.Czech:                "cs",    // cs
+	language.Danish:               "da",    // da
+	language.German:               "de",    // de
+	language.English:              "en",    // en
+	language.AmericanEnglish:      "en",    // en-US
+	language.BritishEnglish:       "en",    // en-GB
+	language.Spanish:              "es",    // es
+	language.EuropeanSpanish:      "es",    // es-ES
+	language.LatinAmericanSpanish: "es",    // es-419
+	language.Finnish:              "fi",    // fi
+	language.French:               "fr",    // fr
+	language.Hebrew:               "he",    // he
+	language.Indonesian:           "id",    // id
+	language.Italian:              "it",    // it
+	language.Japanese:             "ja",    // ja
+	language.Korean:               "ko",    // ko
+	language.Dutch:                "nl",    // nl
+	language.Norwegian:            "no",    // no
+	language.Polish:               "pl",    // pl
+	language.Portuguese:           "pt",    // pt
+	language.BrazilianPortuguese:  "pt-br", // pt-BR
+	language.EuropeanPortuguese:   "pt",    // pt-PT
+	language.Russian:              "ru",    // ru
+	language.Swedish:              "sv",    // sv
+	language.Thai:                 "th",    // th
+	language.Turkish:              "tr",    // tr
+	language.Ukrainian:            "uk",    // uk
+	language.Vietnamese:           "vi",    // vi
+	language.Chinese:              "zh",    // zh
+	language.SimplifiedChinese:    "zh",    // zh-Hans
+	language.TraditionalChinese:   "zh",    // zh-Hant
```

Comment:
These two mappings should now be the same (a general languageTagToCode mapping), so one of them can be removed

Reply:
Ah I see what you mean. Apple is weird and does se for Swedish which meant they aren't the exact same, but we can fix that on the icon side of things. Does feel nicer to have it like this